packages/web-app-files/src/services/folder/spaces/loaderProject.ts

```diff
@@ -74,6 +74,7 @@ export class FolderLoaderSpacesProject implements FolderLoader {
       })
 
       if (!sameRoute) {
+        ref.UPSERT_SPACE(space)
```

Comment:
Doesn't hurt to upsert the space outside of this `if` statement. Even if it's not the displayed space, we'd stell get updated data in the store without extra cost.